deploy/charts/openebs-monitoring/values.yaml

```diff
@@ -120,6 +144,12 @@ kube-prometheus-stack:
       - grafana-polystat-panel
       - snuids-trafficlights-panel
 
+alertRules:
+  pools:
```

Comment:
Instead of pools can it be `cstorPools`? This way readability will increase.

Reply:
Since we are following the naming convention of pools and volumes, I think it is good.